kuma/users/signup.py

```diff
@@ -57,39 +46,8 @@ def __init__(self, *args, **kwargs):
 
     def clean_email(self):
         value = self.cleaned_data["email"]
-        # if the selected email address is "other" we cut things short
-        # and clean the value in the form's clean method instead
-        if value == self.other_email_value:
-            return value
-
-        # otherwise we emmulate the functionality of the EmailField here
-        # stripping whitespaces, validating the content and then
-        # run allauth's own email value cleanup
-        self.cleaned_data["email"] = value.strip()
         validators.validate_email(value)
```

Comment:
This is now the only thing left and it might be that the parent form (i.e. the parent class) already does this and does it better. However, the parent form also messes with checking for dupes on the email which we don't care about in MDN. (...unfortunately, if you ask me)